src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -1,16 +1,19 @@
 <tabset>
   <tab i18n-heading
        heading="OSDs List">
+       
     <cd-table [data]="osds"
               (fetchData)="getOsdList()"
               [columns]="columns"
-              selectionType="single"
+              selectionType="multi"              
```

Comment:
Please remove white spaces at the end of the line.
```suggestion
              selectionType="multi"
```